<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>protocolwindow</class>
 <widget class="QWidget" name="protocolwindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>781</width>
     <height>471</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lbl_protocol">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>25</y>
      <width>511</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Programa 1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_increaseProtocol">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>5</y>
      <width>121</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:3px solid blue;
border-radius:8;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/img/img/button_right.png</normaloff>:/img/img/button_right.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_decreaseProtocol">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>131</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:3px solid blue;
border-radius:8;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/img/img/button_left.png</normaloff>:/img/img/button_left.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>781</width>
      <height>381</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lbl_loadValue">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px solid blue;
border-radius:8;</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>14</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_increaseStage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="img.qrc">
       <normaloff>:/img/img/button_plus.png</normaloff>:/img/img/button_plus.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_decreaseStage">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="img.qrc">
       <normaloff>:/img/img/button_minus.png</normaloff>:/img/img/button_minus.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_increaseLoad">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>150</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="img.qrc">
       <normaloff>:/img/img/button_plus.png</normaloff>:/img/img/button_plus.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_decreaseLoad">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>150</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="img.qrc">
       <normaloff>:/img/img/button_minus.png</normaloff>:/img/img/button_minus.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_loadTag">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:blue;</string>
     </property>
     <property name="text">
      <string>Carga</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_stageTag">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:blue;</string>
     </property>
     <property name="text">
      <string>Estágio</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_secondsTimesFive">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>270</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>x5</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_timeTag">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:blue;</string>
     </property>
     <property name="text">
      <string>Duração</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_exit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>441</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border:3px solid blue;
background-color:#ffffd4;
border-radius:8;</string>
     </property>
     <property name="text">
      <string>Selecionar</string>
     </property>
     <property name="icon">
      <iconset resource="img.qrc">
       <normaloff>:/img/img/button_play_clear.png</normaloff>:/img/img/button_play_clear.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_timeValue">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>50</y>
       <width>151</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px solid blue;
border-radius:8;</string>
     </property>
     <property name="text">
      <string>00:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>14</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_increaseMinutes">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>150</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="img.qrc">
       <normaloff>:/img/img/button_plus.png</normaloff>:/img/img/button_plus.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_decreaseMinutes">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>150</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="img.qrc">
       <normaloff>:/img/img/button_minus.png</normaloff>:/img/img/button_minus.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>58</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_stageValue">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px solid blue;
border-radius:8;</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>14</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_loadTimesFive">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>170</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>x5</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_minutesTag">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>150</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:blue;</string>
     </property>
     <property name="text">
      <string>minutos</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_secondsTag">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>230</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:blue;</string>
     </property>
     <property name="text">
      <string>segundos</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_increaseSeconds">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>230</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="img.qrc">
       <normaloff>:/img/img/button_plus.png</normaloff>:/img/img/button_plus.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_decreaseSeconds">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>230</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="img.qrc">
       <normaloff>:/img/img/button_minus.png</normaloff>:/img/img/button_minus.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>58</height>
      </size>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_minTimesFive">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>190</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>x5</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_loadUnits">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>110</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:blue;</string>
     </property>
     <property name="text">
      <string>Watts</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>lbl_loadValue</zorder>
    <zorder>pb_increaseStage</zorder>
    <zorder>pb_decreaseStage</zorder>
    <zorder>pb_increaseLoad</zorder>
    <zorder>pb_decreaseLoad</zorder>
    <zorder>lbl_loadTag</zorder>
    <zorder>cb_secondsTimesFive</zorder>
    <zorder>pb_exit</zorder>
    <zorder>lbl_timeValue</zorder>
    <zorder>pb_increaseMinutes</zorder>
    <zorder>pb_decreaseMinutes</zorder>
    <zorder>lbl_stageValue</zorder>
    <zorder>cb_loadTimesFive</zorder>
    <zorder>lbl_minutesTag</zorder>
    <zorder>lbl_secondsTag</zorder>
    <zorder>pb_increaseSeconds</zorder>
    <zorder>pb_decreaseSeconds</zorder>
    <zorder>cb_minTimesFive</zorder>
    <zorder>lbl_loadUnits</zorder>
    <zorder>lbl_timeTag</zorder>
    <zorder>lbl_stageTag</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
